rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+    person.rentals << self
+    book.rentals << self
+  end
+
+  attr_accessor :date, :book, :person
+end
```

Comment:
- [ ] Here something similar happens, your `Person` class doesn't have an attribute called `rentals` so this code will fail.

```rb
# person.rb
require_relative 'nameable'
require_relative 'capitalize_decorator'
require_relative 'trimmer_decorator'


class Person < Nameable
  def initialize(age, name = 'Unknown', parent_permission: true)
    super()
    @id = Random.rand(1..1000)
    @name = name
    @age = age
    @parent_permission = parent_permission
  end


  def correct_name
    @name
  end


  attr_reader :id
  attr_accessor :name, :age, :parent_permission


  private


  def of_age?
    @age >= 18
  end


  public


  def can_use_services?
    of_age? || @parent_permission
  end
end
```

Please consider updating your person class in order to handle rentals. 🤓 🔧